<?xml version="1.0"?>
<launch>
	<arg name="TestDevice" value="Spacenav"/>
	<node respawn="true" pkg="spacenav_node" type="spacenav_node" name="$(arg TestDevice)_node">
                <param name="dev" type="string" value="/dev/input/js1" />
                <param name="deadzone" value="0.12"/>
                <!--<param name="autorepeat_rate" value="60"/>-->
        </node>
        <node pkg="masterslave" type="ControlDevice" name="$(arg TestDevice)" args="$(arg TestDevice)" output="screen">
                <param name="button/0" type="string" value="close_gripper"/>
                <param name="button/1" type="string" value="open_gripper"/>
        </node>

	<include file="$(find masterslave)/launch/VisualServoingBasic.launch"/>	

	<node pkg="masterslave" type="NumericKinematic" name="NumericKinematic" output="log"/>
	<node pkg="masterslave" type="NumericKinematicCommander" name="NumericKinematicCommander" args="$(arg TestDevice)" output="log"/>
	<node pkg="masterslave" type="VisualServoing" name="VisualServoing" args="$(arg TestDevice)" output="screen" />
w</launch>
